src/actions/mailing/contactRequestEmail.tsx

```diff
@@ -0,0 +1,30 @@
+'use server'
+import { MailTemplateId, mailersendClient } from '@/lib/mailersendClient'
+import { Recipient } from 'mailersend'
+
+type ContactRequestEmailParams = {
+  senderEmail: string
+  senderFullName: string
+  recipientEmail: string
+  subject: string
+}
+
+export const sendContactRequest = async (
+  cREParams: ContactRequestEmailParams,
```

Comment:
**nit:** It's really hard to guess what it means, let's just do `contactRequestParams` instead of `cREP`. Btw, you can always do:

```ts
export const sendContactRequest = async (
  {senderEmail, senderFullName, subject}: ContactRequestEmailParams,
```

so you don't have to keep that argument name:D

Reply:
Thanks! Implemented.